selfdrive/modeld/models/driving.cc

```diff
@@ -290,13 +291,13 @@ void model_publish(PubMaster &pm, uint32_t vipc_frame_id, uint32_t frame_id,
 
   auto meta = framed.initMeta();
   fill_meta(meta, net_outputs.meta);
-  event.setValid(frame_id < vipc_frame_id + MAX_FRAME_AGE);
+  event.setValid((frame_id < vipc_frame_id + MAX_FRAME_AGE) && sm_alive_valid);
```

Comment:
I would compute frame age only once, and store in a local variable. Same computation is done on line 257.